### DISCUSSION TITLE
is its support external fromwork

### DISCUSSION THREAD

**Discussion by Prashant4900** [supabase/supabase#942] [Questions]
I think Supabase is pretty good if we consider its as a firebase alternative but it also not provide any admin dashboard so I think it's pretty hard for new devs. So my question is it's possible to integrate some pre-build framework or CMS like (Django or Strapi)

**Comment by silentworks** [comment]
I don't think firebase provides any admin dashboard either, but yes one of the benefits of Supabase is that you can connect directly to the Postgres database with any other external framework/cms you choose by using the information found `Settings/Database`. Just make sure that these systems only make use of the `public.` tables. You should never touch anything under `auth.`, `storage.` or `extensions.` directly.
